ci/component.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.23.2-bullseye
+    tag: 1.23.4-bullseye
```

Comment:
This would likely fail [CI](https://concourse.dp-ci.aws.onsdigital.uk/builds/92046167), when adding component test. Other services with component test [use](https://github.com/ONSdigital/dp-frontend-search-controller/blob/ee99348e73d578b41d47bc6ecc6236cc2f4543db/ci/component.yml#L8).

But not part of the this PR

Reply:
There are no component tests. That update should be included in your pr to add them in